File: TwoSum/twoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        ret = [0, 0]
        for i, x in enumerate(nums):
            ret[0] = i
            for i2, x2 in enumerate(nums):
                if i != i2:
                    if x + x2 == target:
                        ret[1] = i2
                        return ret

File: TwoSum/TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] ret = {0, 0};
        for (int j=0; j < nums.length; j++) {
            ret[0] = j;
            for (int i=0; i < nums.length; i++) {
                if(j != i) {
                    if(nums[j] + nums[i] == target) {
                        ret[1] = i;
                        return ret;
                    }
                }
            }
        }
        return ret;
    }
}
